features/webtransport.yml.dist

```diff
@@ -1,15 +1,23 @@
 # Generated from: webtransport.yml
 # Do not edit this file by hand. Edit the source file instead!
 
+status:
+  baseline: false
+  support:
+    chrome: "97"
+    chrome_android: "97"
+    edge: "97"
+    firefox: "114"
+    firefox_android: "114"
 compat_features:
-  # ⬇️ Same status as overall feature ⬇️
   # baseline: false
   # support:
   #   chrome: "97"
   #   chrome_android: "97"
   #   edge: "97"
   - api.WebTransport.createUnidirectionalStream
 
+  # ⬇️ Same status as overall feature ⬇️
```

Comment:
I see the sort order here is actually wrong, the following block has more supporting browsers than the previous. I'll send a PR to fix.

Reply:
That was https://github.com/web-platform-dx/web-features/pull/1295.